### PR TITLE
DE-426,DE-430,DE-431 Extract Product, product families and product price points

### PR DESCRIPTION
**Pull request by maciej-nedza**

### CODE REVIEW

File: components/schemas/Product.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    type:
+      - integer
+      - "null"
+    description: A numerical interval for the length a subscription to this product will run before it expires. See the description of interval for a description of how this value is coupled with an interval unit to calculate the full interval
+  expiration_interval_unit:
+    type:
+      - string
+      - "null"
+    description: "A string representing the trial interval unit for this product, either month or day"
+    enum:
```
[the remark this answers is not in the document]
Comment (michalpierog) [reply components/schemas/Product.yaml:35]: I see there are some places where expiration_interval_unit is just Interval-Unit.yaml
Here it has additional `never` value. So the proper name for this enum can be tricky, Extended-Interval-Unit?

File: components/schemas/Product.yaml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      - integer
+      - "null"
+    description: "The price of the trial period for a subscription to this product, in integer cents."
+  trial_interval:
+    type:
+      - integer
+      - "null"
+    description: A numerical interval for the length of the trial period of a subscription to this product. See the description of interval for a description of how this value is coupled with an interval unit to calculate the full interval
+  trial_interval_unit:
+    type:
```
Comment (michalpierog) [inline components/schemas/Product.yaml:73]: Interval-Unit
You need use oneOf construct:
```
    oneOf:
      - $ref: "./Inteval-Unit.yaml"
      - type: "null"
```

**Review by michalpierog — CHANGES_REQUESTED** [review]

File: components/schemas/List-MRR-Response-Result.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+title: MRR
```
[the remark this answers is not in the document]
Comment (maciej-nedza) [reply components/schemas/List-MRR-Response-Result.yaml:1]: 👀 missed it after copy paste, fixed

**Review by michalpierog — APPROVED** [review]
LGTM

**Pull request merged**